phoenix-pig-base/pom.xml

```diff
@@ -108,6 +108,18 @@
       <type>test-jar</type>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>org.apache.hadoop</groupId>
```

Comment:
Without this the IT tests were complaining they couldn't find the MiniDFSCluster class. Similar dependencies are in the phoenix-hive, etc projects, as is the disabling of the incorrect "dependency not used" error